site/en/docs/privacy-sandbox/private-aggregation/implementation/index.md

```diff
@@ -0,0 +1,461 @@
+    -   K+ frequency measurement
+-   General Usage
+    -   Measure hover-over event inside fenced frames
+    -   Top level navigation
+    -   Controlling where 3rd party can write
+
+### How to view Shared Storage
+
+To view what is stored in Shared Storage, use Chrome DevTools. Stored data can
+be found in `Application -> Shared Storage`.
+
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/wDOc1Bt5ClCICP0Pm98S.png", alt="ALT_TEXT_HERE", width="800", height="462" %}
+
+### View reports for Private Aggregation
+
+To view the aggregatable reports sent, navigate to
+`chrome://private-aggregation-internals`. When debug mode is enabled, a report
+is sent immediately (without a delay) to
+`[[YOUR_ORIGIN]]/.well-known/private-aggregation/debug/report-shared-storage`
+along with the time-delayed report to be sent to
+`[[YOUR_ORIGIN]]/.well-known/private-aggregation/report-shared-storage`.
+
+To enable debugging, follow debugging instructions in the [Enable
+debugging](#enable-debugging) section.
+
+{% Img src="image/RtQlPaM9wdhEJGVKR8boMPkWf443/HWcqZEDSbHeun6qqaQQD.png", alt="ALT_TEXT_HERE", width="800", height="366" %}
+
+## Shared Storage API
+
+To prevent cross-site tracking, browsers have started partitioning all forms of
+storage, including local storage, cookies, and so forth. But there are use cases where unpartitioned storage is required. The Shared Storage API provides unlimited write access across different top-level sites with privacy-preserving read access.
+Shared Storage is restricted to the context origin (the caller of
+`sharedStorage`).
+
+Shared Storage has a capacity limit per origin, with each entry limited to a
+maximum number of characters. If the limit is reached, no further inputs are
+stored. The data storage limits are outlined in the
+[Shared Storage explainer](https://github.com/WICG/shared-storage#data-storage-limits).
+
+### Invoking Shared Storage
+
+Ad techs can write to Shared Storage using JavaScript or response headers. Reading
+from Shared Storage only occurs within an isolated JavaScript environment called
+a worklet.
+
+{% Aside 'key-term' %}
+{% Partial 'privacy-sandbox/glossary-entries/worklet.njk' %}
+{% endAside %}
+
+- **Using JavaScript**
+
+    Ad techs can perform specific Shared Storage functions such as setting, appending,
+    and deleting values outside of a JavaScript worklet. However, specific functions
+    such as reading Shared Storage and performing Private Aggregation have to be
+    completed through a JavaScript worklet.
+
+    Methods that can be used outside of a JavaScript worklet can be found in the
+    [Proposed API Surface - Outside the worklet](https://github.com/WICG/shared-storage#outside-the-worklet).
+
+    Methods that are used in the worklet during an operation can be found in
+    [Proposed API Surface - In the worklet](https://github.com/WICG/shared-storage#in-the-worklet-during-an-operation).
+
+- **Using response headers** (available soon! (~M117))
```

Comment:
Is this available now?

Reply:
This feature will be available in M119 which will become Stable by the end of this month.

Reply:
done